### ISSUE TITLE
captureException is not working - package.json not found

### ISSUE THREAD

**Issue by alxmiron** [getsentry/sentry-electron#135]
When `Sentry.captureException(error)` is called, the error is not tracked because of the error in the console:

```
{ Error: Cannot find module '/Users/aleksey/dev/hitask-web/dist/desktop-testask/package.json'
    at webpackEmptyContext (/Users/aleksey/dev/hitask-web/dist/desktop-testask/main.js:20861:10)
    at getPackageJson (/Users/aleksey/dev/hitask-web/dist/desktop-testask/main.js:71661:39)
    at /Users/aleksey/dev/hitask-web/dist/desktop-testask/main.js:71819:25
    at Generator.next (<anonymous>)
    at fulfilled (/Users/aleksey/dev/hitask-web/dist/desktop-testask/main.js:71586:58)
    at <anonymous> code: 'MODULE_NOT_FOUND' }
```

However `package.json` exists with exactly that path

**Comment by HazAT** [comment]
Fix is on the way

**Comment by timfish** [comment]
We have just released v3.0.0 of the SDK which includes a lot of changes and improvements since this issue was opened. 

If you can still reproduce this with v3, please open a new issue and I'll look into it.
